Makefile

```diff
@@ -0,0 +1,54 @@
+artifact_name       := chips-rest-interfaces-consumer
+version             := "unversioned"
+
+.PHONY: all
+all: build
+
+.PHONY: clean
+clean:
+	mvn clean
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+	rm -f ./build.log
+
+.PHONY: security-check
+security-check:
+	mvn org.owasp:dependency-check-maven:purge
+	mvn org.owasp:dependency-check-maven:check
```

Comment:
On further investigation could you add "-DfailBuildOnCVSS=4" at the end of this line.
This argument will make the make target fail if the CVSS score is too high
As it is, it will pass by default, even if there are vulnerabilities

Reply:
Added the argument - Out of curiosity where did this come from? I could't find any references to it online; are we using this elsewhere?

Reply:
A concourse task was recently added that makes use of this makefile target.
As this is a new project, its a good a place as any to implement it.

The concourse-all-scrum-teams channel is a good place to stay up to date on what is going on with concourse